tools/Replay.hpp

```diff
@@ -174,12 +293,78 @@ struct Stats {
             " server sent " << Util::getHumanizedBytes(_bytesRecvd) <<
             " (" << recvKbps << " kB/s) to " << _connections << " connections.\n";
 
+        _pingLatency.dumpLatencyToCSV("Ping");
+        _tileLatency.dumpLatencyToCSV("Tile");
+
+        dumpStressToCSV(runMs);
+        dumpNetworkStatsToCSV(recvKbps,sentKbps);
```

Comment:
We don't care about the time it took; adding Kbps - adds significant jitter to what -should- be a nice hard number.

Lets log the total _bytesRecvd and _bytesSent - those are the useful things that (I hope) - should be stable =)